doc/index.rst

```diff
@@ -1,8 +1,42 @@
-Plutus
-======
+Plutus Core and Plutus Tx user guide
+==================================================
 
-Plutus provides the support for writing scripts that run on the Cardano blockchain.
-To read more about the overall architecture of writing applications using Plutus, see :ref:`what_is_the_plutus_platform`.
+Plutus Core
+---------------------
+
+The Plutus project consists of Plutus Core, the programming language used for 
+scripts on Cardano; tooling and compilers for compiling various intermediate 
+languages into Plutus Core; and Plutus Tx, the compiler that compiles the Haskell 
+source code into Plutus Core to form the on-chain part of a contract application. 
+All of this is used in combination to write Plutus Core scripts that run on the 
+Cardano blockchain. The Cardano ledger is able to interact with compiled Plutus Core 
```

Comment:
The reference to cardano ledger seems orphaned (could be just me). Some questions come to mind: What is the cardano ledger? How is it different from the blockchain? What is a ledger interface? Why are they mentioned here?

Reply:
My thought was to try to paint an overview picture here. But you may be right that it is misplaced here. I am inclined to remove that statement unless someone else feels otherwise.